examples/typescript/backends/s3/main.ts

```diff
@@ -1,5 +1,6 @@
 import { Construct } from 'constructs';
-import { App, TerraformStack, S3Backend } from 'cdktf';
+import { App, TerraformStack, S3Backend, S3RemoteState } from 'cdktf';
```

Comment:
Terraform only supports one backend. Could you elaborate what is the difference between a `S3Backend` and `S3RemoteState`?